python/torch_mlir/dialects/torch/importer/jit_ir/csrc/import_options.h

```diff
@@ -0,0 +1,19 @@
+//===- import_options.h -----------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// Also available under a BSD-style license. See LICENSE.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef TORCHMLIRJITIRIMPORTER_CSRC_IMPORT_OPTIONS_H
+#define TORCHMLIRJITIRIMPORTER_CSRC_IMPORT_OPTIONS_H
+
+namespace torch_mlir {
+struct ImportOptions {
+  bool assumeTensorHaveValueSemantics = false;
```

Comment:
Can you add a comment here something like:

"""
If this is set to true, then all tensors in the program can be assumed to have value semantics. This can happen, for example, when coming from LazyTensorCore since conversion to value semantics has already happened at a higher level there before we see the program. For calling-convention-impacting decisions, this flag should be interpreted as a requirement to use a value-semantic tensor type (`!torch.vtensor`) in signatures.
"""